pandas/core/dtypes/missing.py

```diff
@@ -33,34 +33,38 @@ def isnull(obj):
 
     Returns
     -------
-    isnulled : array-like of bool or bool
+    isna : array-like of bool or bool
         Array or bool indicating whether an object is null or if an array is
         given which of the element is null.
 
     See also
     --------
-    pandas.notnull: boolean inverse of pandas.isnull
+    pandas.notna: boolean inverse of pandas.isna
+    pandas.isnull: alias of isna
     """
-    return _isnull(obj)
+    return _isna(obj)
 
 
-def _isnull_new(obj):
+isnull = isna
```

Comment:
yes, but the see also section is not fully 'correct' in case of isnull, as it doesn't link to isna (I mean, isna docstring should link to isnull, and isnull docstring to isna. Now they both link to isnull)